src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colours getRandomColor() {
+        int pick = new Random().nextInt(Colours.values().length);
```

Comment:
lets make Random global, because you are creating a new instance of this class each time this method is invoked